rpm/qtimageformats.spec

```diff
@@ -2,7 +2,7 @@ Name:       qt5-qtimageformats
 Summary:    Qt Imageformats
 Version:    5.6.3
 Release:    1%{?dist}
-License:    LGPLv2 with exception or GPLv3
+License:    LGPLv2 with exception or LGPLv3 or Qt Commercial
```

Comment:
I think this should have GPLv3 instead of LGPLv3 still.

Also apparently the files in 3rdparty directory have different licenses (at least BSD) but I guess the combined work is still GPL*?

Reply:
Didn't spot GPLv3 anywhere on the source code headers. Commonly it's:
```
** Alternatively, this file may be used under the terms of the GNU Lesser
** General Public License version 2.1 or version 3 as published by the Free
[...]
```

> Also apparently the files in 3rdparty directory have different licenses (at least BSD) but I guess the combined work is still GPL*?

Think those are commonly unused here as we have the libraries packaged. Jasper not but apparently jpeg2000 doesn't get build here.

Reply:
And even if we built those, I'm guessing the license files shipped by qt could be enough. For comparison haven't spotted fedora digging out the license files deeper.

Reply:
Ah. I checked master branch but the correct one is mer-5.6. Maybe we should switch the default branch on github to avoid future confusion.

Yeah, you seem to be right about this.

Reply:
Btw, the gitmodule points to non-existent url now.

Reply:
Good point on the default branch. Changed that.

The submodule works ok for me.

Reply:
Hmm. Maybe it just doesn't open on browser then.

Reply:
Yeah, it clones fine but I can't browse it on Firefox. Oh well.